pages/docs/data-ops/grns.mdx

```diff
@@ -6,12 +6,24 @@ always remain the same.
 
 ## Anatomy of a GRN
 
-A GRN is comprised of two components, separated by `:`
+A GRN is comprised of three components: type, id, and alias, separated by `:`
```

Comment:
lol https://www.npr.org/2015/03/12/392568604/dont-you-dare-use-comprised-of-on-wikipedia-one-editor-will-take-it-out

(no change needed just funny)